files/ru/web/css/cursor/index.html

```diff
@@ -55,7 +56,7 @@ <h3 id="Значения">Значения</h3>
    </tr>
    <tr style="cursor: default;">
     <td><code>default</code></td>
-    <td><img alt="default.gif" src="/@api/deki/files/3438/=default.gif" style="height: 26px; width: 26px;"></td>
+    <td><img alt="default.gif" src="default.gif"></td>
```

Comment:
There is no image - https://developer.mozilla.org/ru/docs/Web/CSS/cursor/default.gif

Reply:
Only at English version - https://developer.mozilla.org/en-US/docs/Web/CSS/cursor/default.gif

Reply:
So, we can use paths like that `/en-US/docs/Web/CSS/cursor/default.gif`